### PR TITLE
Drop bundler 1x support

### PR DESCRIPTION
**Pull request by xjunior**
Supporting Bundler 1.x is usually troublesome, as we have to do all sorts of gymnastics to deal with the environment variables, lib dependency, and current available/running bundler.

We haven't been using Bundler 1.x for a while in Nitro and other smaller projects.

This PR drops support for bundler 1.x, removing it from our build pipeline.

### This PR also…

- add a `.tool-versions` file
- switches to new power-tools ruby workflow
- checks in all Gemfile.lock (see https://github.com/powerhome/power-tools/pull/100)

### CODE REVIEW

File: .github/workflows/cobra_commander-ruby.yml [hunk 1]
```diff
@@ -5,11 +5,8 @@ on:
 
 jobs:
   ruby:
-    uses: powerhome/power-tools/.github/workflows/_ruby-package.yml@main
+    uses: powerhome/power-tools/.github/workflows/_ruby-workflow.yml@main
     with:
       package: '${{ github.workflow }}'
       workdir: '${{ github.workflow }}'
-      ruby: '["2.7", "3.0", "3.1"]'
```
Comment (benlangfeld) [inline .github/workflows/cobra_commander-ruby.yml:12]: No more testing multiple Ruby versions?

Comment (xjunior) [reply .github/workflows/cobra_commander-ruby.yml:12]: The opposite, it tests more now, just the default for all ruby packages: 2.7, 3.0, 3.1, and 3.2.

**Review by gbmoretti — APPROVED** [review]

**Review by benlangfeld — APPROVED** [review]

**Comment by benlangfeld** [comment]
https://github.com/powerhome/software/blob/main/application.power-tools.tf#L97-L103 to unblock merge.

**Comment by xjunior** [comment]
@benlangfeld https://github.com/powerhome/software/pull/207 Thank you

**Pull request merged**